src/nidigital/metadata/functions.py

```diff
@@ -2048,7 +2048,7 @@
     'Initiate': {
         'codegen_method': 'private',
         'documentation': {
-            'description': 'TBD'
+            'description': 'Starts bursting the pattern, causing the NI-Digital session\nto be committed. To stop the pattern burst, call abort\nmethod. If keep alive pattern is bursting when abort\nis called, keep alive pattern will not be stopped. To stop\nthe keep alive pattern, call abort_keep_alive method.'
```

Comment:
Replace "abort_keep_alive method" with "niDigital_AbortKeepAlive" and "abort method" with "niDigital_Abort".

All public methods should be formatted this way in the metadata. Code generator will replace `niDigital_AbortKeepAlive` with `abort_keep_alive` in the docstring and with hyperlinks in the .rst file.

e.g.:
https://github.com/ni/nimi-python/blob/5dcc7524d37225e4ecf7276da4f83f93b3509885/src/nidigital/metadata/functions_addon.py#L23
translates to https://nimi-python.readthedocs.io/en/master/nidigital/class.html#burst-pattern.

Similarly here's how to format enum values:
https://github.com/ni/nimi-python/blob/5dcc7524d37225e4ecf7276da4f83f93b3509885/src/nidigital/metadata/functions_addon.py#L284

and properties:
https://github.com/ni/nimi-python/blob/5dcc7524d37225e4ecf7276da4f83f93b3509885/src/nidigital/metadata/functions_addon.py#L299